src/Team-Services-Bot.Api/Cards/LogOnCard.cs

```diff
@@ -39,7 +40,7 @@ public LogOnCard(string appId, Uri authorizeUrl, string channelId, string text,
         {
             var button = new CardAction
             {
-                Value = string.Format(UrlOAuth, appId, channelId, userId, Scope, authorizeUrl),
+                Value = string.Format(CultureInfo.InvariantCulture, UrlOAuth, appId, channelId, userId, Scope, authorizeUrl),
```

Comment:
Why not use FormattableString here as well? Must say I didn't know about FormattableString.